[2025-01-10T16:54:04.931+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T16:54:04.944+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:53:00+00:00 [queued]>
[2025-01-10T16:54:04.952+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:53:00+00:00 [queued]>
[2025-01-10T16:54:04.954+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T16:54:04.967+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 16:53:00+00:00
[2025-01-10T16:54:04.973+0000] {standard_task_runner.py:72} INFO - Started process 622 to run task
[2025-01-10T16:54:04.975+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T16:53:00+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmpn9vrhbv9']
[2025-01-10T16:54:04.978+0000] {standard_task_runner.py:105} INFO - Job 101: Subtask append_to_database
[2025-01-10T16:54:05.018+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:53:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T16:54:05.103+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T16:53:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T16:53:00+00:00'
[2025-01-10T16:54:05.108+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T16:54:20.149+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T16:54:20.151+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T16:54:20.159+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T16:54:20.160+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T16:53:00+00:00, execution_date=20250110T165300, start_date=20250110T165404, end_date=20250110T165420
[2025-01-10T16:54:20.183+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T16:54:20.196+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T16:54:20.201+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
